libconnman-qt/networkmanager.cpp

```diff
@@ -900,7 +900,7 @@ void NetworkManager::technologyRemoved(const QDBusObjectPath &technology)
     // if we weren't storing by type() this loop would be unecessary
     // but since this function will be triggered rarely that's fine
     for (NetworkTechnology *net : m_technologiesCache) {
-        if (net->objPath() == technology.path()) {
+        if (net->path() == technology.path()) {
```

Comment:
Don't these get the same path if it exists? `[NetworkTechnology::objPath()](https://github.com/sailfishos/libconnman-qt/blob/master/libconnman-qt/networktechnology.cpp#L281)` just checks if there is technology set and uses `[NetworkTechnology::path()](https://github.com/sailfishos/libconnman-qt/blob/master/libconnman-qt/networktechnology.cpp#L256)` to get the saved path?

Reply:
Ah yeah, memory leak. Forgot to read the PR description. I guess this is ok then.